requirements/runtime.txt

```diff
@@ -1,5 +1,7 @@
+lap
 matplotlib
-numpy
+motmetrics
+numpy==1.23.5
```

Comment:
This can't be written like that. If tracking has to be this version I suggest adding a new txt file.